web/src/components/FailureSwitcher.tsx

```diff
@@ -29,6 +29,7 @@ const FailureSwitcher = ({ error }: PropsType): ReactElement => {
   }, [error])
 
   const getFailureProps = (error: Error): { goToPath?: string; goToMessage?: string; errorMessage: string } => {
+    /* eslint-disable @typescript-eslint/no-unnecessary-condition */
     if (error instanceof NotFoundError && error.city !== undefined && error.language !== undefined) {
```

Comment:
I really don't like disabling linter errors, but I have no clue why this one happens in the first place:

`error.city` and `error.language` *can* be undefined, as per `NotFoundError`'s type. Linter complains that there's no overlap between the first, and each of the last two conditionals.

If someone knows what's up, I'd love to remove the disable-line...

Reply:
You do not need this comment. Linting only fails when you run `yarn lint` in `/web`-subfolder, as the types are from the api-client. If you run linting in parent directory, everything works fine, pipeline should also succeed.

Reply:
Ah, makes sense. Thanks.